src/java.base/share/classes/sun/security/ssl/CipherSuite.java

```diff
@@ -857,6 +853,37 @@
 
     final boolean exportable;
 
+    private static final Map<Integer, CipherSuite> cipherSuiteIds;
+    private static final Map<String, CipherSuite> cipherSuiteNames;
+    private static final List<CipherSuite> allowedCipherSuites;
+    private static final List<CipherSuite> defaultCipherSuites;
+
+    static {
+        Map<Integer, CipherSuite> ids = new HashMap<>();
+        Map<String, CipherSuite> names = new HashMap<>();
+        List<CipherSuite> allowedCS = new ArrayList<>();
+        List<CipherSuite> defaultCS = new ArrayList<>();
+        for(CipherSuite cs : CipherSuite.values()) {
```

Comment:
I may add blank lines before and after the for-loop block.

Reply:
Than you @XueleiFan for approving the PR. 

I have added the black lines before and after the PR.